src/hb-ot-shape.cc

```diff
@@ -642,19 +642,18 @@ hb_ot_rotate_chars (const hb_ot_shape_context_t *c)
     for (unsigned int i = 0; i < count; i++) {
       hb_codepoint_t codepoint = unicode->mirroring (info[i].codepoint);
       if (unlikely (codepoint != info[i].codepoint && c->font->has_glyph (codepoint)))
-	info[i].codepoint = codepoint;
+        info[i].codepoint = codepoint;
       else
-	info[i].mask |= rtlm_mask;
+        info[i].mask |= rtlm_mask;
```

Comment:
Please revert turning tab to spaces, thanks :)

Reply:
It seems to me that the rest of the file is space indented. Or was this meant to be ironic?

Reply:
Rest of the file is mostly consistent on HarfBuzz's current default style that is using two spaces for indention but turning 8 of them to tab, not that fan of the style personally but let preserve it for now :)

Reply:
🤯Wow, I've never seen such a convention, hadn't even noticed that in HarfBuzz. What's the origin of that convention or why would it ever make sense? :) - but yes, updated the CL.

Reply:
@behdad should know :)